### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by brndnb** [FreeCodeCamp/freecodecamp#29]
If free code camp is to be welcoming to all, the website needs more accessibility coverage.

To start, my suggestion is to add an extra bullet on the first challenge (http://www.freecodecamp.com/challenges/0) that makes it clear that we are here to support everyone in their learning with extra information for people that need full accessibility with screen readers or those whom cannot hear the un-captioned challenge videos.

We could have a section with guidelines on what to do if a camper runs into an accessibility issue (seek out someone in the chat, send an email, submit an issue/PR in the repo, etc).

**Comment by QuincyLarson** [comment]
Agreed. If you could put together a list of directions and links for specific accessibilty issues, we can create a special view and send people with accessibility issues there from the first challenge as you suggested (something like freecodecamp.com/accessibility-help).

**Comment by brndnb** [comment]
Great. I'll get started on a rough draft list of current issues that that have been brought to my attention and how we may suggest workarounds at '/accessibility-help.'

Plus, it wouldn't hurt to have an additional challenge on accessibility added to the curriculum eventually. This is super important stuff and way too often overlooked, so teaching it to the fresh campers early on will be a win for the web!

**Comment by QuincyLarson** [comment]
Agreed.


- Quincy Larson
"Anyone who has begun to think, places some portion of the world in
jeopardy." - John Dewey

**Comment by QuincyLarson** [comment]
In the meantime I'm going to close this issue. I've created a Trello card with you on it here: https://trello.com/c/Ulg73Klf
